src/bin/tester.rs

```diff
@@ -163,92 +247,85 @@ fn main() {
             });
         }
         Command::Dir { verbose, dir } => {
-            // Counters for reporting overall stats at the end.
-            let mut total_count = 0;
-            let mut unambiguous_count = 0;
-            let mut ambiguous_count = 0;
-            let mut too_short_count = 0;
-            let mut no_parse_count = 0;
+            // HACK(eddyb) avoid parsing some files that hit
+            // `lykenware/gll` worst-cases (many GBs of RAM usage)
+            // FIXME(eddyb) fix the problems (e.g. implement GC).
+            const BLACKLIST: &[&str] = &[
```

Comment:
Extract out to top level for better visibility (or a separate config file...?)?

Reply:
yea, this one is going to a config file

Reply:
The HACK/FIXME comments were lost when moving to a separate file.

Reply:
this was intentional from my part, now that the blacklist lives externally on the config file, and we have an issue for tracking the memory leak/lack of GC - we can bring that back if you want.